index.js

```diff
@@ -3,16 +3,49 @@ const header = document.querySelector("#header")
 console.log("Here's your header:", header)
 
 
+
 /***** Deliverable 2 *****/
 
+let headerColor = document.getElementById('header')
+headerColor.style.color = 'blue';
```

Comment:
Here, you can use the `header` variable from Deliverable 1 instead (but your code works too)!